### DISCUSSION TITLE
Password encryption/decryption in Collection.

### DISCUSSION THREAD

**Discussion by sunnysonx** [strapi/strapi#8327] [Legacy Topics]
Hello,
I have a Collection: **Websites.**
With Fields: **WebsiteURL**, **Username**, **Password**.

What is the best way to store third party credentials in Strapi?
I know there is a type called Password, but how do I decrypt it? For one time usage in Axios request for login on these websites.
I don't want to store them in .env or something like this, as I also want to offer a possibility to update it periodically.

Thank you!

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
